src/editor/canvas.js

```diff
@@ -127,6 +127,15 @@ export function scaleCanvas(canvas, size, scale = 1) {
   canvas.width = size * scale;
   canvas.height = size * scale;
   const ctx = canvas.getContext('2d');
+  try {
+    // Try to improve the image quality when shrinking large images.
+    if ('imageSmoothingEnabled' in ctx)
+      ctx.imageSmoothingEnabled = true;
+    if ('imageSmoothingQuality' in ctx)
+      ctx.imageSmoothingQuality = 'high';
+  } catch (ex) {
```

Comment:
In fact, I couldn't find it either! 😅

That's why it scared me, and I decided to go fully conservative, trying to prevent a still not met, weird/broken case like "invalid value for property" or "value 'high' is not supported", specially in older mobile browsers.

I originally used that in another project some time ago and I really didn't do too much research on failure cases at the time.

But I think it would be OK to remove the `try/catch` now.

About the `if`'s, indeed, they were a bit pointless back then and also now.

I can close this PR, change that, and submit a new one 😊👍

Reply:
Please remove the `if`s and `try/catch`. Once that's done I'm happy to merge! 

You can also push your changes to this branch to update this PR. That way you won't need to create a new PR

Reply:
Done!

Also, removing the `if`s solved the code style issue! 😅